### ISSUE TITLE
Python workspace interpreter doesn't select virtualenv interpreter

### ISSUE THREAD

**Issue by roblourens** [DonJayamanne/pythonVSCode#1198]
_From @cryst10 on September 8, 2017 5:42_

- VSCode Version: 1.16.0
- OS Version: MacOS Sierra
- Python version: 3.6
- pip: 3
- virtualenv: 3
extension: https://github.com/DonJayamanne/pythonVSCode/wiki

The Python workspace interpreter doesn't select the virtualenv interpreter, it can't even locate it. Also there isn't a possibility of doing it manually, as it wouldn't save. I'm stuck with system wide interpreter so I cannot locate local virtualenv dependencies.

_Copied from original issue: Microsoft/vscode#33992_

**Comment by roblourens** [comment]
Is this an issue with the Python extension? Can you open the issue in that repo if so?

**Comment by roblourens** [comment]
_From @cryst10 on September 8, 2017 5:51_

@roblourens can you forward this issue to that extension like you did with [Microsoft/vscode#33988](https://github.com/Microsoft/vscode/issues/33988) ?

**Comment by DonJayamanne** [comment]
https://github.com/DonJayamanne/pythonVSCode/wiki/Python-Path-and-Version

**Comment by Larz60p** [comment]
I had similar problem on opensuse leap 15.
Here are the steps I used to fix:

I re-installed VSCode one more time and finally everything works.
There are a few things that I did differently, one of these was the answer, but I can't say which, I'm just glad that all is working now:

1. I restored code-runner.executorMap to default values, then changes python entry to:
"python": "$pythonPath $fullFileName"

2. before reloading code, I deleted anything in /tmp related to VSCode.
I am suspecting that this was the part that needed to be done.

3. This is not new as I have been doing it all along, so don't suspect it.
Delete entire .vscode directory in /home/user

I will post this on the VSCode github page as well.
